conmon-rs/server/src/oom_watcher.rs

```diff
@@ -252,8 +253,19 @@ impl OOMWatcher {
                                     };
                                 }
                                 Err(e) => {
-                                    error!("Checking for OOM failed: {}", e);
-                                    match tx.try_send(OOMEvent{ oom: false }) {
+                                    error!("Checking for OOM failed: {:#}", e);
+
+                                    let mut oom = false;
+                                    if !memory_events_file_path.exists() {
```

Comment:
I'm not sure which exists() this is... A better call might be tokio's async call (untested):

```suggestion
 !tokio::fs::metadata("/some/file/path.txt").await?.is_ok()
```

Reply:
Yep, done :+1: